### DISCUSSION TITLE
getting-started-with-mentoring/the-difference-between-mentoring-and-coaching

### DISCUSSION THREAD

**Discussion by giscus** [OfferZen-Community/developer-mentoring#186] [Announcements]
# getting-started-with-mentoring/the-difference-between-mentoring-and-coaching

The differences between a mentor and a coach in their approaches, criteria, and in the dynamic between participants.

https://www.developermentoring.guide/getting-started-with-mentoring/the-difference-between-mentoring-and-coaching

**Comment by marjorieold** [comment]
"While coaching doesn’t have industry standards and oversight" - Actually coaching does have industry standards and oversight. The organization is called International Coaching Federation (ICF). They have very stringent requirements for education and ethics.

**Comment by marjorieold** [comment]
"Coaching is usually short-term"
"it’s the coach’s responsibility to set the agenda for sessions and manage the process" 
I have to respectfully disagree with both of these statements. 
Coaching engagements typically last several months and can go on for years. 
As defined by the ICF, the coachee decides what should be discussed in every coaching session. What you have described as coaching sounds more like consulting.
